Makefile

```diff
@@ -78,6 +86,16 @@ ios:
 	@$(call print, "Building iOS cxframework ($(IOS_BUILD)).")
 	mkdir -p $(IOS_BUILD_DIR)
 	$(GOMOBILE_BIN) bind -target=ios,iossimulator -tags="mobile $(DEV_TAGS) $(RPC_TAGS)" $(LDFLAGS_MOBILE) -v -o $(IOS_BUILD) $(MOBILE_PKG)
+	# modify library files for import without C++ modules
+	sed "s|$(IOS_STRING1)|$(IOS_STRING2)|g" $(IOS_FILE1) > $(IOS_FILE1)-temp
```

Comment:
Could use `sed -i.bak -E 's|$(IOS_STRING1)|$(IOS_STRING2|g' $(IOS_FILE1)` to do the changes inline (with the `-i.bak` being the version of the flags that seems to work on both MacOS and Linux).

Reply:
Works like a charm. Updated.

`-i` by itself was driving me nuts yesterday. Thank you.